Source/Scene/Label.js

```diff
@@ -1196,5 +1249,184 @@ define([
         return false;
     };
 
+    function declareTypes() {
+        var TextTypes = {
```

Comment:
I would just declare this at the top of the file instead of creating it every time.  Then just use `TextTypes` everywhere

Reply:
@mramato If I understand you, you want me to change the code to something like:
```js
var TextTypes = freezeObject({
    LTR : 0,
    RTL : 1,
    WEAK : 2,
    BRACKETS : 3
});
return TextTypes;
```
and put it inside the define near to the glyph functions as is (without a function scope)?

Reply:
Yes, though you won't need the `return` statement.  You can just put the declaration at the top of the file (above `rebindAllGlyphs`), it will be scoped to Label.js.